web-common/static/trace-viewer/index.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
```

Comment:
Subjective, but I wonder if we should put this closer to the observability code, e.g. in `runtime/pkg/observability/embed/file-trace-viewer.html` or `cli/pkg/local/embed/file-trace-viewer.html` and mount it directly on the local server.